main/src/library/MutList.flix

```diff
@@ -456,7 +468,7 @@ namespace MutList {
     pub def copy(v: MutList[a]): MutList[a] & Impure =
         let MutList(a, l) = v;
         let len = deref l;
-        MutList(ref Array.slice(deref a, 0, len), ref len)
+        MutList(ref Array.slice(deref a, 0, capacity(v)), ref len)
```

Comment:
The documentation does state that it is a shallow copy, but I think this was a quick fix to solve the issue of introducing a minimum capacity. Maybe an if-check is required such that the copy is shallow if the length is greater than or equal to the minimum capacity and otherwise a full / deep copy is returned.